host/src/lib/HelperFunctions.jsx

```diff
@@ -271,12 +272,22 @@ export const checkEqualityWithPrevAnswer = (normValue, prevAnswerValue, prevAnsw
   }
 };
 
-export const checkEqualityWithOtherAnswers = (normValue, normType, prevAnswer) => {
+export const checkEqualityWithOtherAnswers = (rawAnswer, normValue, normType, prevAnswer) => {
   // loop through each of the normalized answers in each of the previous answers
+  console.log(normValue, normType, prevAnswer);
   for (let i = 0; i < prevAnswer.normAnswer.length; i++) {
       if (normType === prevAnswer.normAnswer[i].type && checkEqualityWithPrevAnswer(normValue, prevAnswer.normAnswer[i].value, prevAnswer.normAnswer[i].type)) {
         return true;
       }
```

Comment:
these two checks are last ditch and a bit lazy, but they're trying to capture some edge conditions as best as possible that arose from running tests. More info can be found at the later part of this doc:
https://github.com/rightoneducation/righton-app/wiki/Short-Answer-Response-%E2%80%90-Equality-Checks